### ISSUE TITLE
Biomarkers dataset issues

### ISSUE THREAD

**Issue by kmartinez834** [glygener/glygen-issues#135]
Continued from #12 and #13 , the following issues to be addressed in next release:

- The "anatomical_entity" field should match the "uberon_id" field.  Use the misc/sample_mapping.csv file for Uberon mapping.
Currently in unreviewed/glycan_biomarkers.csv:
```
"anatomical_entity","uberon_id"
"DOID:1612 / Breast Cancer","0000178"
```
Should be:
```
"anatomical_entity","uberon_id"
"blood","UBERON:0000178"
```

- Duplicates in human_protein_biomarkers.csv
- Make field names consistent with other datasets

**Blocker for**:
* #474 
* #475 
* #476

**Comment by kmartinez834** [comment]
@jeet-vora

**Comment by kmartinez834** [comment]
Issues included in dataset processing ticket #604
